docs/sources/installation/raspberrypi.md

```diff
@@ -0,0 +1,97 @@
+page_title: Installation on Raspberry Pi
+page_description: Installation instructions for Docker on Raspberry Pi
+page_keywords: raspberry pi, arch linux, hypriot, raspbian, virtualization, docker, documentation, installation
+
+# Raspberry Pi
+
+> **Note:**
+> Docker on the Raspberry Pi is experimental and not officially
+> supported.
+
+There are several ways of installing Docker on the Raspberry Pi.
+
+- Hypriot
+- Raspbian
+- Arch Linux
+
+## Hypriot
```

Comment:
Since this distro was created to run docker and then run a site. I'd prefer just to refer readers to the site. That way, we don't have to keep up with changes on their site to keep our documentation current. 

For the Raspbian installation, putting everything into steps makes it easier to follow. 

--> Markdown: https://gist.github.com/moxiegirl/8e16b35d28951d82fae4

page_title: Installation on Raspberry Pi
page_description: Installation instructions for Docker on Raspberry Pi
page_keywords: raspberry pi, arch linux, hypriot, raspbian, virtualization, docker, documentation, installation

# Raspberry Pi: An experimental install

Docker on the Raspberry Pi is experimental and not officially supported. You can run Docker experimentally on several different Raspberry Pi distribution. This article discusses these distributions:

- Hypriot
- Raspbian
- Arch Linux

## Hypriot

[Hypriot](http://blog.hypriot.com) is a minimal distribution of
[Raspbian](http://raspbian.org) designed to run Docker. Hypriot's version of
Docker works with OverlayFS; the kernel includes the OverlayFS module. Refer to the Hypriot blog for details of using this distribution.

## Raspbian


The [Raspbian](http://raspbian.org)version available on the Raspberry Pi
[download](https://www.raspberrypi.org/downloads/) page does not include Docker.
However, Jessie does.

1. Install Raspbian to a SD card according to the [image installation
guide](https://www.raspberrypi.org/documentation/installation/installing-images/README.md).
2. After the first boot, ensure that the install is current:

		 sudo apt-get update; sudo apt-get -y upgrade

3. Modify the references in `/etc/apt/sources.list` from *wheezy* to *jessie*.
4. Update and upgrade the package managers.

        sudo apt-get update; sudo apt-get upgrade; sudo apt-get dist-upgrade;
	
	You may need to perform this step more than once.

5. Repeat step 4 until there is nothing left to be updated.
6. Install Docker.

		sudo apt-get install docker.io

>**Credit**: These directions are based, in part, on
[Docker, Weave, Raspberry Pi and a bit of Networked Cloud Computing! —
Medium](https://medium.com/@ALGrendel/docker-weave-a-little-cloud-and-a-
raspberry-pi-381f73a4376d).

## Arch Linux

Install Docker on Arch Linux by running `pacman -S docker`.